src/components/Button/index.js

```diff
@@ -133,29 +133,9 @@ const StyledButton = styled.button`
   ${props =>
     props.disabled &&
     css`
-      color: rgba(255, 255, 255, 0.6);
-    `};
-
-  ${props =>
-    props.variant === 'secondary' &&
-    props.disabled &&
-    css`
-      color: ${color.earthLighterAlpha};
-    `};
-
-  ${props =>
-    props.variant === 'caution' &&
-    props.disabled &&
-    css`
-      color: ${color.marsLighterAlpha};
-    `};
-
-  ${props =>
-    props.variant === 'inverted' &&
-    props.disabled &&
-    css`
-      background-color: white;
-      color: ${color.earthLighter};
+      opacity: 0.35;
```

Comment:
I’ll have a look at this with @jelleoverbeek at the office, we might increase this a bit to `0.5`.